src/python/grpcio/grpc/_cython/cygrpc.pyx

```diff
@@ -48,7 +48,7 @@ include "grpc/_cython/_cygrpc/server.pyx.pxi"
 cdef class _ModuleState:
 
   def __cinit__(self):
-    filename = pkg_resources.resource_filename(__name__, '_windows/grpc.def')
+    filename = pkg_resources.resource_filename(__name__, '_windows/grpc_c.64.python')
```

Comment:
No idea how to conditionally select 32 or 64 bits here in that file. I'll leave that to you.